in_toto/hashlib.go

```diff
@@ -0,0 +1,30 @@
+package in_toto
+
+import (
+	"crypto/sha256"
+	"crypto/sha512"
+	"hash"
+)
+
+/*
+getHashMapping returns a mapping from hash algorithm to supported hash
+algorithm.
+*/
+func getHashMapping() map[string]func() hash.Hash {
+	return map[string]func() hash.Hash{
+		"sha256": sha256.New,
+		"sha512": sha512.New,
+		"sha384": sha512.New384,
```

Comment:
Do you think it's worth defining string constants for the algorithm names, given that we use them elsewhere in the code too.

Reply:
Do you have a nice name for it? `sha256String` maybe?

Reply:
Hm. I probably would have used `SHA256` but all-caps does not seem idiomatic for go consts. Let's just keep the strings. :)